src/app/components/MaskedImage/index.tsx

```diff
@@ -25,6 +25,8 @@ const MaskedImage = ({
 
   const isRtl = dir === 'rtl';
 
+  const isBillboard = document.querySelectorAll('[id^="billboard-"]')
```

Comment:
Sorry for snooping, but is it possible to make these changes in the Billboard, rather than touching the masked image component? This is OK to demonstrate the fix in a spike, but I wouldn't want it to be the final solution - a component should not know the context in which it is being used.

Reply:
That's okay, it's actually not working as expected anyway because the change is also affecting the live header despite the conditional logic, so it would need to change because it's clearly not right 😩 I thought this quick hack might work but it didn't, so yes I will move it out of here!